api/queries/getIssues.mdx

```diff
@@ -8,14 +8,15 @@ backlink_title: 'pganalyze GraphQL API Queries'
 
 You can utilize this endpoint to export the currently active issues for a server
 or database, as determined by the pganalyze Check-Up system, in order to export
-them to a third-party system such as PagerDuty.
+them to a third-party system not directly supported by pganalyze's
+[alerting integrations](/docs/checks/#alerting).
 
 ## API
 
 Arguments for `getIssues`:
 
-* **`databaseId` (string)**<br/>Database ID for which to retrieve issues for
-* **`serverId` (string)**<br/>Server ID for which to retrieve issues for (either this or `databaseID` needs to be specified)
+* **`serverId` (string)**<br/>Server ID for which to retrieve issues for
```

Comment:
Just curious: is there any way to mention that this is a "human_id", not UUID? The example kinda demonstrates it but I wonder if further clarification would be helpful

Reply:
Yeah, good question - I think we could probably benefit from a "What's a Server ID" section that describes this in more detail (especially when working with API responses that may contain the raw ID), but given that we mainly use the human IDs in all customer-facing contexts, I think its okay to omit this detail here.